### PR TITLE
Impr/css code and design

### LINKED ISSUE #229: Move inline styles to an external css file.

### PR DESCRIPTION
**Pull request by skkr**
close #229

### CODE REVIEW

**Comment by AugustoL** [comment]
@PabloAlice please review

File: app/src/css/app.scss [hunk 1]
```diff
@@ -20,6 +20,10 @@
   ul li::first-letter {text-transform: uppercase;}
 }
 
+// HOTEL-INFO
+
+#hotel-and-rooms { }
+#hotel-section { }
```
Comment (PabloAlice) [inline app/src/css/app.scss]: ?

Comment (skkr) [reply app/src/css/app.scss]: Is just a helper to keep the structure on the styles organization, no damage IMO. Probably is not included on the resulting css file.

Comment (PabloAlice) [reply app/src/css/app.scss]: ok, I don't know what kind of styles organization you are managing, I don't see any pattern, but if it helps you, it's fine for me.

**Comment by MatiasOS** [comment]
Please, dont merge until we fix an issue with qa instance

File: app/src/css/app.scss (hunk 1 above)
Comment (skkr) [reply app/src/css/app.scss]: Thanks! Styles are arranged from most global to most specific, by section.

**Pull request merged**